### PR TITLE
CLOUDP-73728: mongocli ops-manager admin backup s3 lists

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-73728

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
./bin/mongocli om admin backup s3 list -P ops
ID                         URI                         SSL     LOAD FACTOR   S3 AUTH METHOD
andrea.angiolillo.bucket   mongodb://127.0.0.1:27017   false   50            KEYS
```

```
./bin/mongocli om admin backup s3 list -P ops -o json
{
  "links": [
    {
      "rel": "self",
      "href": "http://localhost:8080/api/public/v1.0/admin/backup/snapshot/s3Configs?pageNum=1\u0026itemsPerPage=100"
    }
  ],
  "results": [
    {
      "id": "andrea.angiolillo.bucket",
      "uri": "mongodb://127.0.0.1:27017",
      "writeConcern": "ACKNOWLEDGED",
      "ssl": false,
      "assignmentEnabled": true,
      "encryptedCredentials": false,
      "loadFactor": 50,
      "awsAccessKey": "XXXXXXXXXX",
      "awsSecretKey": "XXXXXX",
      "s3AuthMethod": "KEYS",
      "s3BucketEndpoint": "s3-eu-west-1.amazonaws.com",
      "s3BucketName": "andrea.angiolillo.bucket",
      "s3MaxConnections": 50,
      "disableProxyS3": true,
      "acceptedTos": true,
      "sseEnabled": false,
      "pathStyleAccessEnabled": false
    }
  ],
  "totalCount": 1
}
```

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
waiting for https://github.com/mongodb/mongocli/pull/486

File: internal/cli/opsmanager/admin/backup/s3/list.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var listTemplate = `ID	URI	SSL	LOAD FACTOR	S3 AUTH METHOD{{range .Results}}
```
Comment (gssbzn) [inline internal/cli/opsmanager/admin/backup/s3/list.go:26]: [non blocking] could be just AUTH METHOD given the scope of the command but I don't feel strong about it
```suggestion
var listTemplate = `ID	URI	SSL	LOAD FACTOR	AUTH METHOD{{range .Results}}
```

**Review by gssbzn — APPROVED** [review]
LGTM

File: internal/cli/opsmanager/admin/backup/s3/list.go (hunk 1 above)
Comment (andreaangiolillo) [reply internal/cli/opsmanager/admin/backup/s3/list.go:26]: I am going to change this in the next command

**Pull request merged**